src/ai/backend/client/cli/session.py

```diff
@@ -270,6 +267,13 @@ def create(
         assigned_agent_list = assign_agent
         with Session() as session:
             try:
+                if name is None:
+                    result = session.ComputeSession.paginated_list(fields=[session_fields["name"]])
+                    names = [item["name"] for item in result.items]
+                    if (name := get_random_name()) in names:
+                        name = duplicated_name(name)
```

Comment:
It is a good idea to duplication check here, but we do not need to stay session name as unique.
And this loop iterates all the existing session names.